rowboat/plugins/utilities.py

```diff
@@ -420,27 +462,29 @@ def info(self, event, user):
         created_dt = to_datetime(user.id)
         content.append('Created: {} ago ({})'.format(
             humanize.naturaldelta(datetime.utcnow() - created_dt),
-            created_dt.isoformat()
+            created_dt.strftime("%d-%b-%y @ %H:%M")
         ))
 
+        if rdb.sismember('global_admins', user.id):
+            content.append(u'\n**\u276F Heimdallr Staff**')
```

Comment:
There should be a `ROWBOAT_NAME`  param someplace

Reply:
fixed